store/s3store.go

```diff
@@ -134,12 +129,12 @@ func (s *S3Store) Write(id SecretId, value string) error {
 
 	putObjectInput := &s3.PutObjectInput{
 		Bucket:               aws.String(s.bucket),
-		ServerSideEncryption: aws.String(s3.ServerSideEncryptionAes256),
+		ServerSideEncryption: types.ServerSideEncryptionAes256,
 		Key:                  aws.String(objPath),
 		Body:                 bytes.NewReader(contents),
 	}
 
-	_, err = s.svc.PutObject(putObjectInput)
+	_, err = s.svc.PutObject(context.TODO(), putObjectInput)
```

Comment:
Shouldn't the same context be passed down by the CLI to each of these methods? Sorry if I'm incorrect with my assumptions here about the way context is supposed to work.

Reply:
Does it work? By that I mean if you press Ctrl+C while it's doing something does it properly handle it / exit?

Reply:
If this wasn't something that worked before it's fine to leave it as is.